manifests/p/Prusa3D/PrusaSlicer/2.5.0/Prusa3D.PrusaSlicer.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Prusa3D.PrusaSlicer
+PackageVersion: 2.5.0
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /quiet
+  SilentWithProgress: /passive
+UpgradeBehavior: install
+Installers:
+- Architecture: x86

```

I’ve not checked the logs but could it be this?
```suggestion
- Architecture: x64
```